north/src/runtime/linux/mod.rs

```diff
@@ -43,5 +43,52 @@ pub async fn init(config: &Config) -> Result<()> {
     debug!("Entering mount namespace");
     sched::unshare(sched::CloneFlags::CLONE_NEWNS).context("Failed to enter mount namespace")?;
 
+    // Pipe minijail log to the runtime
+    capture_minijail_log().await?;
+
+    Ok(())
+}
+
+async fn capture_minijail_log() -> Result<()> {
+    use async_std::{os::unix::io::FromRawFd, stream::StreamExt};
+    use futures::io::AsyncBufReadExt;
```

Comment:
We should avoid using `futures`. This crate will be removed with the tokio migration because it's reported by `tokio` and `async-std`. No idea why we added it back then. Think there is a `AsyncBufReadExt` in `async-std` also.

Reply:
`futures` is a dependency of `async-std`. In this case, it seems it is what `async-std` is using to get the iterator from `lines()`. I didn't want to put `use async_std::prelude::*`.

Reply:
Why not? We're using it so the use is fine. Instead of the wildcard we could also explicitly import what we need -> `BufReadExt`

Reply:
that's what I did, i just found that it was a reexport of `futures::io::AsyncBufReadExt`. In the latest change this is replaced with `async_std::io::prelude::BufReadExt`.